core.py

```diff
@@ -21,6 +22,7 @@ def __init__(self, questions, search_engines, storage, maxlength,
 		self.search = Search(search_engines, parent=self)
 		self.pingpong = PingPong(parent=self)
 		self.flooding = Flooding(maxlength, paste, parent=self)
+		self.lutcha = Lutcha(parent=self)
```

Comment:
lutcha ist keine Kern-Komponente, deshalb gehört das auch nicht in `core.py`, sondern in `complementary.py`.